src/wasm/cpp/test-particle.cc

```diff
@@ -225,5 +225,54 @@ class SimpleParticle : public arcs::Particle {
   arcs::Info local_;
 };
 
-DEFINE_PARTICLE(TestParticle)
 DEFINE_PARTICLE(SimpleParticle)
+
+
+class ServiceParticle : public arcs::Particle {
```

Comment:
This is a really cool example. Thanks for choosing to convert this service area to cpp wasm.